packages/react/README.md

```diff
@@ -15,6 +15,20 @@ $ yarn add @untool/react react react-dom react-router-dom react-helmet
 # OR npm install @untool/react react react-dom react-router-dom react-helmet
 ```
 
+## `render(element, [options])`
+
+`render()` is `@untool/react`'s main export. You are expected to call it in your applications main entry file and it is essentialy a shorthand: it creates and bootstraps a core mixin container and calls its `render` method.
```

Comment:
"... creates and bootstraps a *core* mixin container..." this should be reworded to indicate that it is not a mixin container of the type **core**.